### PR TITLE
fix: proposal page alignment & spacing

### PR DESCRIPTION
**Pull request by sahas-01**
Spacing and alignment issues fixed in proposal page across different screen sizes.

### CODE REVIEW

File: src/components/Post/PostHeading.tsx [hunk 1]
```diff
@@ -84,15 +85,18 @@ const PostHeading: FC<IPostHeadingProps> = (props) => {
 								<PostSummary className='hidden md:flex' />
 								: null
```
Comment (KanishkaRajputd) [inline src/components/Post/PostHeading.tsx:86]: try to do it with only one <PostSummary component

File: src/ui-components/CreationLabel.tsx [hunk 2]
```diff
@@ -105,9 +105,10 @@ const CreationLabel: FC<ICreationLabelProps> = (props) => {
 			<div className='flex items-center text-lightBlue'>
 				{(topic || text || created_at) && <>
 				&nbsp;
-					<Divider className='ml-1 hidden md:inline-block' type="vertical" style={{ borderLeft: '1px solid #485F7D' }} />
+					<Divider className={`ml-1 md:inline-block ${!isRow ? 'hidden' : 'inline-block'}`} type="vertical" style={{ borderLeft: '1px solid #485F7D' }} />
 				</>}
-				{created_at && <span className='flex items-center pl-5 mt-2 md:pl-0 md:mt-0'><ClockCircleOutlined className='mx-1' />{relativeCreatedAt}</span>}
+				{created_at && <span className={`flex items-center md:pl-0 mr-1 ${isRow ? 'mt-0' : 'xs:mt-2 md:mt-0'}`}><ClockCircleOutlined className='mr-1' />{relativeCreatedAt}</span>}
+				{/* <Divider className={'ml-1 mt-2 xs:inline-block md:hidden'} type="vertical" style={{ borderLeft: '1px solid #485F7D' }} /> */}
```
Comment (KanishkaRajputd) [inline src/ui-components/CreationLabel.tsx:111]: please remove commented code

**Review by KanishkaRajputd — COMMENTED** [review]
please resolved changes requested

File: src/ui-components/CreationLabel.tsx (hunk 2 above)
Comment (sahas-01) [reply src/ui-components/CreationLabel.tsx:111]: Resolved, pls check

**Pull request merged**